API.md

```diff
@@ -363,6 +363,17 @@ LoRa.enableInvertIQ();
 
 LoRa.disableInvertIQ();
 ```
+### LNA Gain
+
+Set LNA Gain for better RX sensitivity, by default ADC is used and LNA Gain is not used.
+
+```arduino
+LoRa.setGain(gain);
```

Comment:
@IoTThinks I forgot this last time but `keywords.txt` also needs a `setGain` line (note, this file must use tabs).

Reply:
Put in the keywords.txt.
https://github.com/sandeepmistry/arduino-LoRa/pull/408/commits/d4953a67327960122feaebf9630de93fbed47c13